packages/app-content-pages/src/screens/Team/TeamContainer.js

```diff
@@ -30,8 +31,9 @@ function TeamContainer (props) {
 TeamContainer.getInitialProps = async ({ req }) => {
   const host = getHost(req)
   let error
-  const teamData = await request.get(host + '/api/team')
-    .then(res => res.body)
+  let teamData
```

Comment:
```suggestion
  let teamData = []
```
It's good practice to set a default here.

Reply:
thanks - OOI what about the `let error` above, I assume as that will be undefined instead of used in the mapping functions it's ok to not have a default?

Reply:
Depends how consuming code is supposed to detect errors, I guess. It looks like you return both `teamData` and `error`. Will anything bad happen if both are defined at the same time?

Reply:
no - it handles the missing data (original error). I've added mappable default for teamData